lib/physics/box.go

```diff
@@ -79,7 +91,12 @@ func (b *Box) Contains(position coords.World) bool {
 
 func (b *Box) inSolid(world mapgen.BlockSource) bool {
 	blockCollide := func(x, y, z int) bool {
-		return world.Block(coords.Block{x, y, z}).Solid()
+		blockCoord := coords.Block{x, y, z}
+		if world.Block(blockCoord).Solid() {
+			blockBox := NewBox(blockCoord.Center().Vec3(), vmath.Vec3{0.5, 0.5, 0.5})
+			return b.Collides(blockBox)
+		}
+		return false
```

Comment:
Wait a second, what was the point of this change? It seems slower and more complex, and I'm not sure the benefit (apologies if we discussed this already and I just forgot)